test/test_helper.exs

```diff
@@ -4,3 +4,13 @@ ExUnit.start()
 unless Version.compare(System.version(), "1.8.0") == :lt do
   ExUnit.after_suite(&Cleanup.execute/1)
 end
+
+credentials =
+  "GCP_CREDENTIALS"
+  |> System.fetch_env!()
+  |> Base.decode64!()
```

Comment:
why this line?
I'm running this:
```sh
export WAFFLE_BUCKET=...
export GCP_CREDENTIALS=$(cat ~/service-account.json)
mix test
```

and from `Goth`'s docs it doesn't include it so I'm curious if it's just an alternative format I wasn't aware of or something else?

Reply:
ahh yeah, it is not the official format. A colleague shared with me encoded in 64 to be easier to select and copy paste. I will remove